src/sql/workbench/parts/dashboard/widgets/properties/media/propertiesWidget.css

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+body {
+	/* styling likely inherited, but calling it out here since we have none in codepen */
+	font-family: "Segoe UI";
+	font-size: 13px;
+	font-weight: 400;
+}
+
+
+.flex-table-row {
+	display: flex;
+	margin: 0;
+	padding: 0;
+}
+
+.flex-table-cell,
+.flex-table-cell-destination {
+	box-sizing: border-box;
+	flex-grow: 1;
+	/* width: 50%; */
+	padding: 8px;
+	overflow: hidden;
+	list-style: none;
+	border-top: solid 1px #ccc;
+	background: #fff;
+	font-size: 12px;
+}
+
+.flex-table-cell {
+	font-weight: 600;
+	/* styling to match Portal */
+	width: 30% !important;
+}
+
+.flex-table-cell-destination {
+	width: 70% !important;
+}
+
+.flex-table-cell-destination a {
+	color: rgb(1, 92, 218) !important;
+	text-decoration: none;
+}
+
+a:hover {
+	text-decoration: underline;
+}
+
+.darktheme .flex-table-cell,
```

Comment:
I don't believe .darktheme is valid. We use pattern:
```css
.vsdark .flex-table-cell
```
and also duplicate for high contrast using `.hc-black` selector.

Reply:
Ohh, I took Andrew's help for css and he mailed me this. I'll check and make the appropriate changes.